source/connect.txt

```diff
@@ -38,133 +37,118 @@ This is equivalent to the following command:
 
    mongosh "mongodb://localhost:27017"
 
-Local MongoDB Instance on a Non-Default Port
---------------------------------------------
+Local Deployment on a Non-Default Port
+--------------------------------------
 
-To specify a port to connect to on localhost, you can use:
+To specify a port to connect to on localhost, you can use either:
 
-- A :manual:`connection string </reference/connection-string>`. 
+- A :manual:`connection string </reference/connection-string>`.
 
-  .. example::
-  
-     To connect to a MongoDB instance running on localhost with a 
-     non-default port 28015:
-  
-     .. code-block:: sh
-  
-        mongosh "mongodb://localhost:28015"
+- The ``--port`` command-line option.
 
-- The command-line option ``--port``.
+For example, the following commands connect to a deployment running on
+localhost port 28015:
 
-  .. example:: 
+.. code-block:: sh
 
-     To connect to a MongoDB instance running on localhost with a 
-     non-default port 28015:
+   mongosh "mongodb://localhost:28015"
 
-     .. code-block:: sh
+.. code-block:: sh
 
-        mongosh --port 28015
+   mongosh --port 28015
 
-MongoDB Instance on a Remote Host
----------------------------------
+MongoDB Deployment on a Remote Host
```

Comment:
[suggestion] We removed "MongoDB" from other headings -- should we remove it here?